File: maratona_de_programacao/trabalho1/7.py
n = list(map(float, input().split()))

m1 = (n[0]*2 + n[1]*3 + n[2]*4 + n[3])/10

print(f"Media: {m1:.1f}")
if m1 >= 7:
    print("Aluno aprovado.")
elif m1 < 5:
    print("Aluno reprovado.")
else:
    print("Aluno em exame.")
    ne = float(input())
    print(f"Nota do exame: {ne:.1f}")
    nf = (ne + m1) / 2
    if nf >= 5:
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print(f"Media final: {nf:.1f}")

File: maratona_de_programacao/trabalho1/7.c
#include <stdio.h>

int main() {
    double n1, n2, n3, n4;
    double m1, ne, nf;

    scanf("%lf %lf %lf %lf", &n1, &n2, &n3, &n4);

    m1 = (n1 * 2 + n2 * 3 + n3 * 4 + n4) / 10;

    printf("Media: %.1f\n", m1);

    if (m1 >= 7.0) {
        printf("Aluno aprovado.\n");
    }
    else if (m1 < 5.0) {
        printf("Aluno reprovado.\n");
    }
    else {
        printf("Aluno em exame.\n");

        scanf("%lf", &ne);

        printf("Nota do exame: %.1f\n", ne);

        nf = (ne + m1) / 2;

        if (nf >= 5.0) {
            printf("Aluno aprovado.\n");
        }
        else {
            printf("Aluno reprovado.\n");
        }

        printf("Media final: %.1f\n", nf);
    }

    return 0;
}